python/setup.py

```diff
@@ -76,7 +76,6 @@ def finalize_options(self):
         "Programming Language :: C",
         "Programming Language :: C++",
         "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.5",
```

Comment:
also change `readme.md` to remove 3.5

Reply:
Done. Also added ML support and GPU support to initial list of core features.